src/create-styles.ts

```diff
@@ -50,7 +50,7 @@ function toStyleObject(css: string): Style {
   for (const rule of stylesheet.rules) {
     if (rule.type === `rule` && `selectors` in rule) {
       for (const selector of rule.selectors || []) {
-        const utility = selector.replace(/^\./, ``).replace(`\\/`, `/`);
+        const utility = selector.replace(/^\./, ``).replace(`\\`, `/`);
```

Comment:
```suggestion
        const utility = selector.replace(/^\./, ``).replace(`\\`, ``);
```
I think the correct fix is to replace double backslashes with _nothing_.

Reply:
Stupid me! Did that!